modules/substrate/src/lib.rs

```diff
@@ -151,6 +153,8 @@ decl_error! {
 		Halted,
 		/// The pallet has already been initialized.
 		AlreadyInitialized,
+		/// The given header is not a decendent of a particular header.
+		NotDecendent,
```

Comment:
```suggestion
		/// The given header is not a descendent of a particular header.
		NotDescendent,
```
?

Reply:
Good catch, you were [close](https://grammarist.com/usage/descendant-descendent/) (and closer than I was, lol).